### ISSUE TITLE
Can't open some components from Private mode

### ISSUE THREAD

**Issue by trmdi** [AdguardTeam/AdguardBrowserExtension#2435]
### AdGuard Extension version

4.1.57

### Browser version

Microsoft Edge 115.0.1901.203

### OS version

Windows 11

### What filters do you have enabled?

No filters

### Issue Details

When you have allowed Adguard to work in the private mode, in that mode, when you click on Adguard > Setting/Open filtering log... Nothing will happen.

**Comment by maximtop** [comment]
I've tested it on my end and couldn't reproduce the issue.
@Birbber could you check please too?

**Comment by trmdi** [comment]
> I've tested it on my end and couldn't reproduce the issue. @Birbber could you check please too?

Are you opening the DevTools? If so, close that window before trying reproducing it.

**Comment by maximtop** [comment]
> Are you opening the DevTools? If so, close that window before trying reproducing it.
Nope, I've only allowed the extension to work in private mode. Open a new incognito tab and check the filtering log.

**Comment by maximtop** [comment]
I was finally able to replicate the issue. On macOS, there isn't an option to start in incognito mode. However, if you close the regular window, the filtering log doesn't open.

Thank you, we will see what we can do.

**Comment by trmdi** [comment]
Today when I open Adguard setting from the incognito tab, it can be opened in a new normal browser window (not incognito). Can it be opened in the current incognito window? uBlock can do that then it can not be an Edge API limitation.

**Comment by maximtop** [comment]
It seems like this - `However, if you close the regular window, the filtering log doesn’t open.` - was a Chromium bug that can no longer be reproduced.
